gr-network/lib/udp_source_impl.h

```diff
@@ -53,7 +52,8 @@ class NETWORK_API udp_source_impl : public udp_source
 
     // A queue is required because we have 2 different timing
     // domains: The network packets and the GR work()/scheduler
-    boost::circular_buffer<char>* d_localqueue;
+    gr::buffer_sptr d_localqueue_writer;
```

Comment:
these should be `unique_ptr`, as far as I can tell.

Sadly, `gr::make_buffer` returns a shared pointer (it should really probably just return an object by value, C++17 guarantees return value optimization; the original gr::make_buffer code predates std:: smart pointers in GR, however, I think.).

So, leave as is for now :/